tensorflow/python/ops/image_ops_test.py

```diff
@@ -1592,6 +1593,29 @@ def testContrastFactorShape(self):
                                 "Shape must be rank 0 but is rank 1"):
       image_ops.adjust_contrast(x_np, [2.0])
 
+  @test_util.run_gpu_only
+  @test_util.run_all_in_graph_and_eager_modes
+  def testDeterminismUnimplementedExceptionThrowing(self):
+    config.enable_deterministic_ops(True)
```

Comment:
You should also disable deterministic ops at the end of the test, by putting the entire method in a try-finally block and disabling it in the finally block. E.g.:

```python
try:
  config.enable_deterministic_ops(True)
  ... # Rest of test
finally:
  config.enable_deterministic_ops(False)
```

Reply:
Well spotted. The next commit factors this is into `test_util.deterministic_ops`.